ui/package.json

```diff
@@ -1,91 +1,95 @@
-{
-  "name": "ui",
-  "version": "1.0.0",
-  "description": "the service mesh management plane",
-  "main": "index.js",
-  "scripts": {
-    "dev": "eslint . && nodemon ui_dev_server.js",
-    "lint": "eslint --fix .",
-    "next": "next",
-    "build": "next build",
-    "start": "NODE_ENV=production node ui_dev_server.js",
-    "export": "next export",
-    "cy:run": "cypress run",
-    "cy:open": "cypress open",
-    "test": "start-server-and-test dev http://localhost:3000 cy:run && eslint ."
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint",
-      "pre-push": "npm run lint",
-      "...": "..."
-    }
-  },
-  "keywords": [],
-  "author": "Layer5",
-  "repository": "https://github.com/layer5io/meshery",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "@date-io/moment": "^1.3.13",
-    "@fortawesome/fontawesome-svg-core": "^1.2.30",
-    "@fortawesome/free-brands-svg-icons": "^5.14.0",
-    "@fortawesome/free-regular-svg-icons": "^5.14.0",
-    "@fortawesome/free-solid-svg-icons": "^5.14.0",
-    "@fortawesome/react-fontawesome": "^0.1.8",
-    "@material-ui/core": "^4.9.12",
-    "@material-ui/icons": "^4.5.1",
-    "@material-ui/lab": "^4.0.0-alpha.56",
-    "@material-ui/pickers": "^3.2.8",
-    "@zeit/next-css": "^1.0.1",
-    "billboard.js": "=1.11.0",
-    "caniuse-lite": "^1.0.30001109",
-    "classnames": "^2.2.6",
-    "codemirror": "^5.56.0",
-    "eslint-loader": "^4.0.2",
-    "immutable": "^4.0.0-rc.12",
-    "isomorphic-unfetch": "^3.0.0",
-    "js-yaml": "^3.13.1",
-    "jsonlint-mod": "^1.7.4",
-    "jss": "latest",
-    "minimist": ">=1.2.2",
-    "moment": "^2.27.0",
-    "mui-datatables": "^3.3.1",
-    "next": "^9.1.5",
-    "next-compose": "0.0.2",
-    "next-redux-wrapper": "^3.0.0-alpha.3",
-    "notistack": "^0.8.2",
-    "prop-types": "latest",
-    "react": "^16.12.0",
-    "react-codemirror2": "^5.1.0",
-    "react-countdown-clock": "^2.6.0",
-    "react-dom": "^16.12.0",
-    "react-jss": "^10.3.0",
-    "react-lazyload": "^2.6.5",
-    "react-moment": "^0.8.4",
-    "react-redux": "^6.0.1",
-    "react-select": "^2.4.3",
-    "redux": "^4.0.4",
-    "redux-devtools-extension": "^2.13.8",
-    "redux-thunk": "^2.3.0",
-    "styled-jsx": "latest",
-    "uuid": "^8.3.0"
-  },
-  "devDependencies": {
-    "@babel/traverse": "^7.9.6",
-    "@babel/types": "^7.11.0",
-    "babel-eslint": "^10.1.0",
-    "cypress": "^5.4.0",
-    "eslint": "^7.6.0",
-    "eslint-config-airbnb": "^18.2.0",
-    "eslint-config-google": "^0.14.0",
-    "eslint-plugin-cypress": "^2.11.2",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
-    "eslint-plugin-react": "^7.20.5",
-    "eslint-plugin-react-hooks": "^4.0.8",
-    "http-proxy": "^1.18.1",
-    "husky": "^4.2.5",
-    "nodemon": "^2.0.4",
-    "start-server-and-test": "^1.11.2"
-  }
-}
+{

```

Comment:
Why there is a complete rewrite of the package.json? Can you please try to fix it and push only the relevant changes?

Reply:
Else it looks good!

Reply:
This was my second attempt at it. Not sure why it's showing complete rewrite 🤔

Reply:
I guess this is the issue with your editor configuration, try once disabling your editor formatting else let's go ahead with it

Reply:
FIxed in #1805